src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -1,5 +1,7 @@
 // eslint-disable-file import/no-extraneous-dependencies
-import React, { useState, useEffect, useRef } from 'react';
+import React, {
```

Comment:
the changes in this `Embedding.jsx` file seem to be much more than just shuffling points. Can you explain what is happening here?